.gitignore

```diff
@@ -0,0 +1 @@
+deps/replicated
```

Comment:
What generates this?

Reply:
Hey, I have a Makefile that creates this, so I added it to the ignore, thanks! 👍